src/main/java/com/openelements/conduct/integration/github/GitHubWebhookHandler.java

```diff
@@ -0,0 +1,104 @@
+package com.openelements.conduct.integration.github;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnBean;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.openelements.conduct.data.CodeOfConductProvider;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@ConditionalOnBean(GitHubCodeOfConductProvider.class)
+public class GitHubWebhookHandler {
```

Comment:
I would like to remove this class for now. I understand the idea and like it. General I would like to only have 1 GitHub weebhook endpoint. Since we already have GitHubWebhookEndpoint the functionality should be integrated in that endpoint. With that ConditionalOnBean can not be used but a programatic check for the bean must happen. With all that I prefare to remove it for now and do it in a future PR